hoomd_organics/base/simulation.py

```diff
@@ -640,18 +912,54 @@ def run_displacement_cap(
         self.operations.updaters.remove(std_out_logger_printer)
 
     def temperature_ramp(self, n_steps, kT_start, kT_final):
+        """Create a temperature ramp.
+
+        Parameters
+        ----------
+        n_steps : int; required
+            The number of steps to ramp the temperature over.
+        kT_start : float; required
+            The starting temperature.
+        kT_final : float; required
+            The final temperature.
+
+        """
         return hoomd.variant.Ramp(
             A=kT_start, B=kT_final, t_start=self.timestep, t_ramp=int(n_steps)
         )
 
     def pickle_forcefield(self, file_path="forcefield.pickle"):
+        """Pickle the list of HOOMD forces.
```

Comment:
We could add a sentence about the use cases for this method, like restarting a simulation, or running a different simulation with the same system (e.g. Running NVT, then switching over to a Tensile simulation)